packages/conf-binutils/conf-binutils.master/files/find-binutils.ml.in

```diff
@@ -0,0 +1,228 @@
+open Printf
+open StdLabels
+
+type file = {
+  path : string;
+  dgst : string;
+}
+
+type tool = {
+  test  : string -> bool;
+  main  : file option;
```

Comment:
I think that we decided to drop the `main::files` separation and just provide one variable. Is this a reminiscent of the old design?

Reply:
it's kind of convenient to have this `main`. e.g. in case user wants to override the default search and provide a path to a preferred tool manually. If we add it to a list as any other path, we'll need to maintain order in the list of paths.